corehq/apps/app_manager/suite_xml.py

```diff
@@ -1913,6 +1914,44 @@ def get_target_module(case_type, module_id, with_product_details=False):
             except IndexError:
                 pass
 
+        self.add_parent_datums(datums, module)
+
+        return datums, assertions
+
+    def add_parent_datums(self, datums, module):
```

Comment:
We use the term datum as a name of an element and not as the singular of
data, hence datums.
On 3 Jul 2015 18:15, "Norman Hooper" <notifications@github.com> wrote:

> In corehq/apps/app_manager/suite_xml.py
> <https://github.com/dimagi/commcare-hq/pull/7246#discussion_r33872801>:
>
> > @@ -1913,6 +1914,44 @@ def get_target_module(case_type, module_id, with_product_details=False):
> >              except IndexError:
> >                  pass
> >
> > +        self.add_parent_datums(datums, module)
> > +
> > +        return datums, assertions
> > +
> > +    def add_parent_datums(self, datums, module):
>
> [image: :smiley:] datums bothered me too. But I left it the way I found
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/7246/files#r33872801>.
>